includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -2250,4 +2242,30 @@ public static function maybe_keep_variations( $delete_variations, $product, $fro
 
 		return $delete_variations;
 	}
+
+	/**
+	 * Check if the current page is the Edit Subscription page
+	 *
+	 * @return bool True if the current page is the Edit Subscription page
+	 *
+	 * @since 7.5.0
+	 */
+	private static function is_edit_subscription_page() {
+		if ( ! is_admin() || ! function_exists( 'get_current_screen' ) ) {
+			return false;
+		}
+
+		$screen = get_current_screen();
+		if ( ! is_object( $screen ) ) {
+			return false;
+		}
+
+		if ( wcs_is_custom_order_tables_usage_enabled() ) {
+			$subscriptions_page_id = 'woocommerce_page_wc-orders--shop_subscription';
+		} else {
+			$subscriptions_page_id = 'shop_subscription';
+		}
+
+		return $subscriptions_page_id === $screen->id;
```

Comment:
We can use the `wcs_get_page_screen_id()` helper function here. It will will get the expected screen id for the edit `shop_subscription` screen. 

```suggestion
		return wcs_get_page_screen_id( 'shop_subscription' ) === $screen->id;
```

Reply:
Oh, that's great to know! Thanks